src/nft/components/collection/MarketplaceSelect.tsx

```diff
@@ -46,6 +48,7 @@ const MarketplaceItem = ({
       removeMarket(value)
       setCheckboxSelected(false)
     }
+    sendAnalyticsEvent(EventName.NFT_FILTER_SELECTED, { marketplace: value, checked: !isCheckboxSelected })
```

Comment:
Did you run into any ordering issues here? This runs after the `setCheckboxSelected` so on paper it doesn't seem like you would need to negate the `isCheckboxSelected` unless you called the analytics event before the set method

Reply:
true, it doesn't seem like it should be necessary but somehow, even after the set method, negation was necessary to get the right value passed (same for comment below).